addon/components/challenges/add.js

```diff
@@ -54,10 +115,21 @@ export default Ember.Component.extend(ClickOutside, {
     saveDismiss(){
       this.actions.save.bind(this)();
       this.actions.dismiss.bind(this)();
+    },
+    selectMajor(){
```

Comment:
Initially didn't do that because I though that choosing major/minor should be an active choice and toggling both when they have the same value would work in that case
I can make the initial choice minor, then I can toggle